config/crd/patches/cainjection_in_kappcontrollerconfigs.yaml

```diff
@@ -0,0 +1,8 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+# CRD conversion requires k8s 1.13 or later.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
+  name: kappcontrollerconfigs.run.tanzu.vmware.com
```

Comment:
We don't need to submit this file for now

Reply:
I think it's fine to include this file. It is commented out in `config/crd/kustomization.yaml ` and not enabled. We can reenable these files when we need them.

Reply:
^ What @maralavi said. Lets add later when webhook stuff is ready.